Adding Examples (and some other stuff)

milljm opened the pull request:
This commit adds examples to our moose docs.

Renaming the 'Getting Started' navbar to 'Install', as Examples will
now reside in 'Getting Started'. This is an attempt to adhere to the
common way most other sites name their items in the their navbar.

Closes #10345

Commit:
Adding Examples (and some other stuff)

This commit adds examples to our moose docs.

Renaming the 'Getting Started' navbar to 'Install', as Examples will
now reside in 'Getting Started'. This is an attempt to adhere to the
common way most other sites name their items in the their navbar.

Closes #10345

sapitts commented on docs/content/documentation/modules/index.md:
```diff
@@ -2,19 +2,80 @@
+---
+
+Available in: [moose/modules/solid_mechanics](https://github.com/idaholab/moose/tree/devel/modules/solid_mechanics)
+
```
Would you be willing to exchange tensor mechanics for the solid mechanics in this example please? Just so that we can point new users to just tensor mechanics as we slowly work on replacing solid mechanics

Commit:
Remove unicode characters

Closes #10345

moosebuild commented on docs/content/media/large_media:
```diff
@@ -1 +1 @@
-Subproject commit ebd77065e811529edb75ce21525a955a0c5b7263
+Subproject commit f5bb586cf526ec0226a858e4b48836453e44b9d7
```
**Caution!** This contains a submodule update

moosebuild commented:
Job [Precheck](https://www.moosebuild.org/job/140573/) on d01bafc wanted to post the following:

Warning: This PR changes repo size by 11.24 MiB.

moosebuild commented:
Job [Documentation](https://www.moosebuild.org/job/140572/) on d01bafc wanted to post the following:

View the site [here](http://mooseframework.org/docs/PRs/10385/site)

This comment will be updated on new commits.

Commit:
Fixing unittests, file size rejections, etc

Altering the mention of solid_mechanics to tensor_mechines
per @sapitts request.

Include a CSS change that clears any floating div object
for '##' headings. This allows what ever is between two
'##' headings (H2) to not overlap with the next.

This helps when dealing with images with the float
attribute.

See:

Getting Started, Examples, Example 01

With out this change images like mug.e would flow into
the 'Vaiables' section.

Closes # 10345

moosebuild commented on docs/content/media/large_media:
```diff
@@ -1 +1 @@
-Subproject commit ebd77065e811529edb75ce21525a955a0c5b7263
+Subproject commit a2c2df7cd14ed66ec0d66eb1890866d0c116a8ca
```
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild replied:
**Caution!** This contains a submodule update

moosebuild commented:
Job [Precheck](https://www.moosebuild.org/job/140585/) on 0c1b612 wanted to post the following:

Warning: This PR changes repo size by 10.03 MiB.

Commit:
Fixing equations for Preconditioning page

Fixing equations so they render...

Closes # 10345

moosebuild commented:
Job [Precheck](https://www.moosebuild.org/job/141119/) on c8e118c wanted to post the following:

Warning: This PR changes repo size by 10.03 MiB.

aeslaughter commented:
@milljm Why does this have a libMesh update?

aeslaughter commented on docs/content/getting_started/additional_information/dissertations_and_theses_working_using_moose_software.md:
```diff
@@ -0,0 +1,22 @@
+[](name: publications)
+[](.tiny[.scroll[)
```
These can be removed

aeslaughter commented on docs/content/documentation/systems/Preconditioning/index.md:
```diff
@@ -1,6 +1,315 @@
+`-sub_pc_type` | `ilu, lu, hypre` | Can be used with bjacobi or asm
+`-pc_hypre_type` | `boomeramg` | Algebraic multigrid
+`-pc_hypre_boomeramg` (cont.) | | "Information Threshold" for AMG process
+` _strong_threshold` | `0.0 - 1.0` | **(0.7 is auto set for 3D)
```
remove **, this will try to make text bold

aeslaughter commented on docs/content/documentation/modules/index.md:
```diff
@@ -2,19 +2,80 @@
     _No_ export controlled physics (e.g., neutronics) should be put into the modules.
+
+## Modules Anatomy
+---
```
How are your find and replace skills? I would like all these "---" to be removed. If we want a horizontal line we should do it with css rather than making manual lines like this.

milljm replied:
@aeslaughter How do we do this in CSS, in markdown? I was looking for an example, but I can't find one in our markdown style guide page.

aeslaughter replied:
I don't think we need the lines at all, I wouldn't worry about it.

permcody commented:
Closing per the 30-day policy

Commit:
Fixing syntax/addressing comments

Refs #10345

moosebuild commented:
Job [Precheck](https://www.moosebuild.org/job/154571/) on e67eee7 wanted to post the following:

Warning: This PR changes repo size by 10.03 MiB.

permcody commented:
Closing due to inactivity.
